CHANGES.rst

```diff
@@ -5,6 +5,9 @@ Changes for Crate Admin Interface
 Fixes
 -----
 
+- Fixed an issue that prevented the value for nested partitioned columns to
+  show up in the table partitions overview.
```

Comment:
"columns showing up in the..." would be more correct. the use of "prevented" earlier in the sentence means that this part of the sentence should also use the past tense

alternatively, you could change the tense to present indicative (which is probably the better option):

"Fixed an issue that prevents the value for nested partitioned columns showing up in the table partitions overview."